spec/requests/organizer_can_create_event_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe 'POST /events' do
+  let(:user) { create(:user) }
+  let!(:group_1) { create(:group) } 
+
```

Comment:
Minor thing but no need to call it ":group_1" as there is only one group being created for the test. Rename to ":group"